scripts/ask_adult.sh

```diff
@@ -0,0 +1,22 @@
+# код языка
+#!/bin/bash 
+# вывод в командную строку сообщения
+echo -n "Введите значение переменной a  " 
+# запись ввода из командной строки в переменную а
+read a
```

Comment:
I guess that variable should have a proper name. You can use `age`, for example.